catalog/imaging/imaging-app/src/main/resources/features.xml

```diff
@@ -16,6 +16,12 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
+    <feature name="imaging-render-plugin" install="auto" version="${project.version}"
```

Comment:
that decision was made long ago.  The idea would be anything image related would go into this app and not be restricted to just NITFs (I'm thinking more support for things like GeoTIF, etc). This specific feature should probably just be called `nitf-render-plugin` as it does appear specific to the data type.